config/config.go

```diff
@@ -17,6 +17,7 @@ type Config struct {
 	MongoDBURL      string `env:"MONGODB_URL"                           flag:"mongodb-url"                     flagDesc:"MongoDB server URL"`
 	Database        string `env:"INSOLVENCY_MONGODB_DATABASE"           flag:"mongodb-database"                flagDesc:"MongoDB database for data"`
 	MongoCollection string `env:"INSOLVENCY_MONGODB_COLLECTION"         flag:"mongodb-collection"              flagDesc:"The name of the mongodb collection"`
+	IsEfsAllowListAuthDisabled bool `env:"DISABLE_EFS_ALLOW_LIST_AUTH"         flag:"disable-efs-allow-list-auth"              flagDesc:"Set to 'true' in order to bypass EFS allow list aspect of API authorisation"`
```

Comment:
Please run `goimports` for alignment

Reply:
Have now configured `goimports` on save in my IDE for Golang (turns out I had Prettier set as default in VS Code - which was unable to do any relevant linting/formatting).